website_docs/exporters.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Exporters
+weight: 4
+---
```

Comment:
Add an alias entry for `exporting_data`.

Reply:
done